Junk4Dollars-app/ShowAuctions/Auction.swift

```diff
@@ -0,0 +1,6 @@
+struct Auction {
+    var title: String
+    var description: String
+    var starting_price: Int
```

Comment:
I believe the convention is camelCase. e.g. `startingPrice` instead of `starting_price`